charts/ccsm-helm/templates/_helpers.tpl

```diff
@@ -17,7 +17,15 @@ app.kubernetes.io/name: {{ template "ccsm.name" . }}
 helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 app.kubernetes.io/managed-by: {{ .Release.Service }}
-app.kubernetes.io/version: {{ .Chart.AppVersion }}
+{{- if .Values.image }}
+    {{- if .Values.image.tag }}
+app.kubernetes.io/version: {{ .Values.image.tag | quote }}
+    {{- else }}
+app.kubernetes.io/version: {{ .Values.global.image.tag | quote }}
+    {{- end }}
+{{- else }}
+app.kubernetes.io/version: {{ .Values.global.image.tag | quote }}
+{{- end }}
```

Comment:
Just for reference for how to find the right template in regards to `{{- }}`

* https://pkg.go.dev/text/template#hdr-Text_and_spaces
* Playground https://go.dev/play/p/5rMAdr_kD1x

Reply:
:wrench: I think the logic here is a bit redundant, right? If I understand correctly, it could just be
```
{{- if .Values.image.tag }}
app.kubernetes.io/version: {{ .Values.image.tag | quote }}
{{- else }}
app.kubernetes.io/version: {{ .Values.global.image.tag | quote }}
{{- end }}
```

Reply:
Unfortunately not. Since the .Values.image can be null in the parent chart, which caused failures.